pkgs/stdenv/linux/default.nix

```diff
@@ -13,6 +13,7 @@
       "x86_64-linux" = import ./bootstrap-files/x86_64.nix;
       "armv5tel-linux" = import ./bootstrap-files/armv5tel.nix;
       "armv6l-linux" = import ./bootstrap-files/armv6l.nix;
+      "armv7a-linux" = import ./bootstrap-files/armv7l.nix;
```

Comment:
I think this would also require the bootstrap tarball in question.

Reply:
They're identical instruction sets. It should be safe to use these bootstrap tools for any version of armv7. In fact, I would argue that we shouldn't even provide armv7 bootstrap and instead use armv5 for all arm32 bootstrapping.